docs/source/ryzenai/overview.mdx

```diff
@@ -22,6 +22,8 @@ Note:
 The RyzenAI Model requires a runtime configuration file. A default version of this runtime configuration file can be found in the Ryzen AI VOE package, extracted during installation under the name `vaip_config.json`.
 For more information refer to [runtime-configuration-file](https://ryzenai.docs.amd.com/en/latest/runtime_setup.html#runtime-configuration-file)
 
+In case no runtime configuration file is provided, the library will use the configuration defined in RyzenAIXXX model class. For available configs see [ryzenai/configs/](https://github.com/huggingface/optimum-amd/tree/main/optimum/amd/ryzenai/configs/).
```

Comment:
```suggestion
In case no runtime configuration file is provided, the library will use the configuration defined in RyzenAIXXX model class. For available configurations, please refer to [ryzenai/configs/](https://github.com/huggingface/optimum-amd/tree/main/optimum/amd/ryzenai/configs/).
```

Reply:
Done